system/Language/en/Time.php

```diff
@@ -1,21 +1,57 @@
 <?php
 
+/**
+ * CodeIgniter
+ *
+ * An open source application development framework for PHP
+ *
+ * This content is released under the MIT License (MIT)
+ *
+ * Copyright (c) 2014-2018 British Columbia Institute of Technology
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @package	CodeIgniter
+ * @author	CodeIgniter Dev Team
+ * @copyright	2014-2018 British Columbia Institute of Technology (https://bcit.ca/)
+ * @license	https://opensource.org/licenses/MIT	MIT License
+ * @link	https://codeigniter.com
+ * @since	Version 3.0.0
+ * @filesource
+ */
+
 return [
-	'invalidMonth' => 'Months must be between 0 and 12.',
-	'invalidDay' => 'Days must be between 0 and 31.',
-	'invalidHours' => 'Hours must be between 0 and 23.',
-	'invalidMinutes' => 'Minutes must be between 0 and 59.',
-	'invalidSeconds' => 'Seconds must be between 0 and 59.',
-	'years' => '{0, plural, =1{# year} other{# years}}',
-	'months' => '{0, plural, =1{# month} other{# months}}',
-	'weeks' => '{0, plural, =1{# week} other{# weeks}}',
-	'days' => '{0, plural, =1{# day} other{# days}}',
-	'hours' => '{0, plural, =1{# hour} other{# hours}}',
-	'minutes' => '{0, plural, =1{# minute} other{# minutes}}',
-	'seconds' => '{0, plural, =1{# second} other{# seconds}}',
-	'ago' => '{0} ago',
-	'inFuture' => 'in {0}',
-	'yesterday' => 'Yesterday',
-	'tomorrow' => 'Tomorrow',
-	'now' => 'Just now',
+	'invalidMonth' 		=> 'Months must be between 0 and 12.',
+	'invalidDay' 		=> 'Days must be between 0 and 31.',
+	'invalidHours' 		=> 'Hours must be between 0 and 23.',
+	'invalidMinutes' 	=> 'Minutes must be between 0 and 59.',
+	'invalidSeconds' 	=> 'Seconds must be between 0 and 59.',
+	'years' 			=> '{0, plural, =1{# year} other{# years}}',
```

Comment:
These should be spaces used for alignment. As you can see here on GitHub, use tabs for alignment doesn't always work :)